hash/msx2_flop.xml

```diff
@@ -12436,6 +12466,34 @@ The following floppies came with the machines.
 		</part>
 	</software>
 
+	<software name="saotomeb">
+		<description>Blue Wind (Japan)</description>
+		<year>1990</year>
+		<publisher>studio ANGEL</publisher>
+		<info name="alt_title" value="早乙女学園ブルーウィンド"/>
```

Comment:
Should be “Saotome Gakuen Blue Wind”.

Reply:
I had initially chosen for 'Blue Wind' since that is what it says on the front of the box. Updated it to 'Saotome Gakuen Blue Wind' which is what it says on the side of the box.